src/language-js/print/array.js

```diff
@@ -2,16 +2,18 @@
 
 const { printDanglingComments } = require("../../main/comments");
 const {
-  builders: { line, softline, group, indent, ifBreak },
+  builders: { line, softline, group, indent, ifBreak, fill },
 } = require("../../document");
 const { getLast } = require("../../common/util");
 const {
   shouldPrintComma,
   hasComment,
   CommentCheckFlags,
   isNextLineEmpty,
+  isNumericLiteral,
 } = require("../utils");
 
+const { hardline } = require("../../document/doc-builders");
```

Comment:
nit: TIL I learned prettier uses semicolons!

surprised! havent seen those in ages ;).

Reply:
;)